### PR TITLE
cli: improve orcid CLI logging

### PR DESCRIPTION
**Pull request by ammirate**
This commit changes also the integration tests for `import_legacy_orcid_tokens`, by making the tests style compliant to the rest of the project.

* INSPIR-2594

Signed-off-by: Antonio Cesarano <cesarano2607@gmail.com>

### CODE REVIEW

File: backend/inspirehep/orcid/cli.py [hunk 1]
```diff
@@ -18,6 +25,49 @@ def orcid():
...
+                        },
+                    )
+            else:
+                secho("Cannot link user and token.", fg='yellow')
+
+        except SQLAlchemyError as ex:
+            secho(str(ex), fg='red')
+
+    secho('No more data to process.')
+    db.session.commit()
```
Comment (drjova) [inline backend/inspirehep/orcid/cli.py]: just a detail, weird spacing :)

File: backend/tests/integration/orcid/test_orcid_cli_import_legacy_tokens.py [hunk 2]
```diff
@@ -191,22 +236,27 @@ def test_import_multiple_orcid_tokens_no_user_exists(
     assert_db_has_n_legacy_tokens(0, SAMPLE_USER_2)
 
     # Migrate
-    import_legacy_orcid_tokens()
+    result = app_cli_runner.invoke(import_legacy_orcid_tokens)
+
+    # both authors are not found
+    output = result.output.split('\n')
+    assert output.count('No row was found for one()') == 2
```
Comment (drjova) [inline backend/tests/integration/orcid/test_orcid_cli_import_legacy_tokens.py]: do we care about the output? just checking the app if the tokens are there and the queue is empty is not enough?

File: backend/inspirehep/orcid/cli.py (hunk 1 above)
Comment (drjova) [reply backend/inspirehep/orcid/cli.py]: Does it make sense to remove the logs? We can use them for stats etc. WDYT

File: backend/tests/integration/orcid/test_orcid_cli_import_legacy_tokens.py (hunk 2 above)
Comment (ammirate) [reply backend/tests/integration/orcid/test_orcid_cli_import_legacy_tokens.py]: I didn't want to change the test logic, that's why I kept them. Some tests are actually focusing on the code throwing exceptions.

File: backend/inspirehep/orcid/cli.py (hunk 1 above)
Comment (ammirate) [reply backend/inspirehep/orcid/cli.py]: I think the CLI itself doesn't really provide any interesting statistic compared to the actual pushing tasks, those are important to observe.

Comment (ammirate) [reply backend/inspirehep/orcid/cli.py]: @drjova could you be more specific about the spacing? :P

Comment (drjova) [reply backend/inspirehep/orcid/cli.py]: for example spacing between else and except

Comment (ammirate) [reply backend/inspirehep/orcid/cli.py]: @drjova updated

**Pull request merged**